docs/parameters.yaml

```diff
@@ -1327,6 +1327,177 @@ default: true
 components: ["origin", "director", "registry"]
 ---
 ############################
+#   Shoveler-level configs   #
+############################
+name: Shoveler.Enable
+description: >-
+  Enable XRootD monitoring shoveler: https://github.com/opensciencegrid/xrootd-monitoring-shoveler
+type: bool
+default: false
+components: ["origin", "cache"]
+---
+name: Shoveler.MessageQueueProtocol
+description: >-
+  Select which protocol to use in order to connect to the MQ. Options are amqp, stomp
+
+  For amqp, the following configurations are required:
+   - URL: amqps://username:password@example.com/vhost
+   - Exchange: shoveled-xrd
+   - Topic: mytopic
+   - TokenLocation: /etc/pelican/xrootd-monitoring-shoveler-token
+
+  For stomp, the following configurations are required:
+   - User: username
+   - Password: password
+   - URL: messagebroker.org:port
+   - Topic: mytopic
+   - Cert: path/to/cert/file
+   - CertKey: path/to/certkey/file
+type: string
+default: amqp
+components: ["origin", "cache"]
+---
+name: Shoveler.URL
+description: >-
+  For amqp and stomp.
+
+  The URL to connect to the shoveler
+type: url
+default: none
+components: ["origin", "cache"]
+---
+name: Shoveler.Exchange
```

Comment:
I'd suggest that for protocol-specific configurations, you prefix the sub variable. Eg `Shoveler.AMQPExchange`, `Shoveler.StopUsername`